### PR TITLE
Delete the dubtrack module

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

idk if I should do something with the api wrappers file, but dubtrack's site appears to be fully down

### CODE REVIEW

**Comment by RAnders00** [comment]
Why don't we just remove it. The dubtrack domain isn't even registered anymore. If they ever come back, thanks to git, we can restore it.

File: pajbot/apiwrappers/README.md [hunk 1]
```diff
@@ -271,8 +271,6 @@ In the above examples, we never handled the case where the user we were looking
 - **Return `None`** for methods that strictly _get_ data and where the absence of data is not generally an error case, but can happen normally (like for example `get_user`) - or e.g. in the case of the BTTV API wrapper - when a channel has no emotes
 - **Raise an exception** in all other cases, and in cases where the non-existance of data either means the method cannot continue normally.
 
-Take for example a method that seeks to fetch the currently playing song on Dubtrack - Say we first query the API for the current song, and then we query the API again in another call for the song's title, author, youtube link, etc. The second API call looks up the song using the internal ID the song is given by Dubtrack (which we got in the first API call's response) - in this case, getting a 404 Not Found error in the second API call would be an error condition. Getting the 404 Not Found error in the first stage (e.g. nothing currently playing), then the correct thing would be to return `None`.
-
```
Comment (RAnders00) [inline pajbot/apiwrappers/README.md:275]: why did you remove this. It's still a valid explanation and example

Comment (alazymeme) [reply pajbot/apiwrappers/README.md:275]: How is that a valid example? If the module doesn't exist, then how can people reference the error handling?

Comment (alazymeme) [reply pajbot/apiwrappers/README.md:275]: @pajlada What would be the most ideal module to use as an example here?

Comment (RAnders00) [reply pajbot/apiwrappers/README.md:275]: The point is, the explanation doesn't even reference the code. It's explained in a way that can be fully understood without looking into the dubtrack apiwrapper implementation. although we could just add a code block that contains that particular piece of code

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,11 +2,15 @@
...
 
+- Minor: Removed the dubtrack module (#916)
+
 ## v1.45
 
 Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` when updating to this version!
 
 - Major: Added a new config option `whisper_output_mode`, allowing you to send whispers to the streamer's chat or to disable them altogether, to aid un-verified bots that may be unable to send whispers. View the example config for more info. (#878)
+- Minor: Added an extra config option for _known_ bots (in addition to verified bots that were already supported). (#878)
+- Minor: Moved chat alerts into a sub-category. Also added a new raid alert module
```
Comment (RAnders00) [inline CHANGELOG.md:13]: There seems to be some merge fuck-up here

Comment (RAnders00) [reply CHANGELOG.md:13]:
```suggestion
```
